src/scripts/MultiPicklist.js

```diff
@@ -0,0 +1,325 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import FormElement from './FormElement';
+import Icon from './Icon';
+import Button from './Button';
+import { default as DropdownMenu, DropdownMenuItem } from './DropdownMenu';
+import { isElInChildren } from './util';
+import uuid from 'uuid';
+
+
+export default class MultiPicklist extends Component {
```

Comment:
Can't we use the original picklist with new prop?

Reply:
No